NCR-Corporation/sample-app-burgers
# Removing Login/Logout/Register UI, Code Cleanup

## Description
This PR hides the Login/Logout/Register UI elements from across the site.
It also does some code cleanup and organization:
1. Files have been formatted with `prettier`
2. The `..menu.html` files are now extending `base.html` instead of including the JS and menu every time. This will reduce chances of missing changes made to the header/navigation.
3. Fixing of error on homepage of button on submit function not found

## Screenshots/Screencasts
See: https://recordit.co/opi4Mo5Jpf (hopefully that works)

## Testing
1. Check out this branch and start the site
2. Search for stores
3. Visit each location and confirm filtering, add to cart, etc. still works
4. Visit the cart
5. Confirm adding items, removing items from cart still works
6. Checkout
7. Confirm taken to confirmation screen
8. Visit about page
9. Confirm documentation in about still works

## Ticket
#15
